src/smc-webapp/project_store.ts

```diff
@@ -286,21 +294,6 @@ export class ProjectStore extends Store<ProjectStoreState> {
       }
     },
 
-    active_file_sort: {
-      fn: () => {
-        if (this.getIn(["active_file_sort"]) != null) {
-          return this.getIn(["active_file_sort"]).toJS();
-        } else {
-          const is_descending = false;
-          const column_name = (this.redux.getStore("account") as any).getIn([
-            "other_settings",
-            "default_file_sort"
-          ]);
-          return { is_descending, column_name };
-        }
-      }
-    },
-
```

Comment:
This relied on `getIn([arg])` and `get(arg)` having two different behaviors. Definitely an abstraction leak. See #4325.